setup.py

```diff
@@ -8,20 +8,28 @@
 from setuptools import setup, find_packages
 
 
-def get_version():
+def load_version():
     """Returns the version found in nilearn/version.py
```

Comment:
This docstring is not accurate any more. Something like "Executes nilearn/version.py in a globals dictionary and return it'